dev-requirements.txt

```
@@ -1,17 +1,17 @@
-black==24.10.0; python_version > "3.8"
-black==24.8.0; python_version <= "3.8"
+black==25.1.0; python_version > "3.8"
+black==25.1.0; python_version <= "3.8"
```

```suggestion
black==24.10.0; python_version > "3.8"
black; python_version <= "3.8"
```
We may just want to take the 3.8 stuff out at this point, but for now I'm just taking the version off so it won't get auto-updated (similar to what we're doing with flake8 below)

Also, bandit update is changing files so stay with 24.10.0 for now until those can also be updated.